src/main/java/com/rcv/ContestConfig.java

```diff
@@ -386,6 +386,11 @@ private void validateRules() {
         isValid = false;
         Logger.log(Level.SEVERE, "batchElimination can't be true in a multi-winner contest!");
       }
+    } else {
+      if (isHareQuotaEnabled()) {
+        isValid = false;
+        Logger.log(Level.SEVERE, "hareQuota can't be true in a single-seat contest!");
```

Comment:
Slight change in wording since this will trigger if `getNumberOfWinners()` is null or <= 1.
```suggestion
        Logger.log(Level.SEVERE, "hareQuota can only be true in a multi-seat contest!");
```